src/crypto/Crypto.cpp

```diff
@@ -237,13 +239,23 @@ namespace Crypto
 {
     bool init()
     {
-        if (Botan::version_major() != 2 || Botan::version_minor() < 11) {
+#ifdef KPXC_DEV_BOTAN3
+        if (Botan::version_major() != 3) {
+            g_cryptoError = QObject::tr("Botan library must be at least 3.x, found %1.%2.%3")
```

Comment:
Instead of creating a new translation string (and duplicating code), change the 2.11.x and 3.x to be an injection:

`QObject::tr("Botan library must be at least %1, found %2.%3.%4").arg(minBotanVersion).arg(....`

Reply:
I switched to using the `ifdef/else/endif` to define the major version, min minor version, and version string.